service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/C2UploadedEmailContentProvider.java

```diff
@@ -25,16 +24,16 @@
     private final ObjectMapper mapper;
     private final Time time;
 
-    public Map<String, Object> buildC2UploadNotification(final CaseDetails caseDetails) {
+    public C2UploadedTemplate buildC2UploadNotification(final CaseDetails caseDetails) {
```

Comment:
Minor: I know it was here before but `buildC2UploadNotificationTemplate()` is perhaps better?

Reply:
will change :)